src/main/java/com/asusoftware/transporter/service/impl/EmployeeServiceImpl.java

```diff
@@ -37,6 +40,27 @@ public Employee findById(UUID id) {
     return employeeRepository.findById(id).orElseThrow(EmployeeNotFoundException::new);
   }
 
+  @Override
+  public void update(UUID employeeId, UpdateEmployeeDto updateEmployeeDto) {
+    Employee employee = employeeRepository.findById(employeeId).orElseThrow(EmployeeNotFoundException::new);
+    employee.setFirstName(updateEmployeeDto.getFirstName());
+    employee.setLastName(updateEmployeeDto.getLastName());
+    employee.setEmail(updateEmployeeDto.getEmail());
+    employee.setPhone(updateEmployeeDto.getPhone());
+    employee.setRole(updateEmployeeDto.getRole());
+    employee.setAddress(Optional.ofNullable(updateEmployeeDto.getAddress()).map(AddressDto::toAddress).orElse(employee.getAddress()));
+    employeeRepository.save(employee);
+  }
+
+  @Override
+  public void removeEmployee(RemoveEmployeeDto removeEmployeeDto) {
+    Employee employee = findById(removeEmployeeDto.getEmployeeId());
+    Company company = companyService.findById(removeEmployeeDto.getCompanyId());
+    if (employee.getCompany().equals(company)) {
```

Comment:
o logica in plus si inutila... de ce am nevoie sa stiu si compania daca eu doar sterg angajatul?